src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -438,12 +438,22 @@ private void testGetCoursesForStudentAccount() throws Exception {
         assertEquals(2, courseList.size());
 
         CourseAttributes course1 = dataBundle.courses.get("typicalCourse1");
-        assertEquals(course1.getId(), courseList.get(0).getId());
-        assertEquals(course1.getName(), courseList.get(0).getName());
 
         CourseAttributes course2 = dataBundle.courses.get("typicalCourse2");
-        assertEquals(course2.getId(), courseList.get(1).getId());
-        assertEquals(course2.getName(), courseList.get(1).getName());
+
+        boolean isCourse1Present = false;
+        boolean isCourse2Present = false;
+
+        for (CourseAttributes course : courseList) {
+            if (course.getId().equals(course1.getId()) && course.getName().equals(course1.getName())) {
+                isCourse1Present = true;
+            } else if (course.getId().equals(course2.getId()) && course.getName().equals(course2.getName())) {
+                isCourse2Present = true;
+            }
```

Comment:
Unnecessary; the list size is checked and the list content is checked via the two boolean vars.